malib/agent/agent_interface.py

```diff
@@ -310,6 +310,7 @@ def request_data(
 
         while batch is None:
             batch, info = ray.get(self._offline_dataset.sample.remote(buffer_desc))
+            # print(">>>>>> sample info:", info)
```

Comment:
maybe we should delete the debug output code, although you comment on it?

Reply:
sure